TODO.md

```diff
@@ -4,10 +4,10 @@
 
 # jos: sähköposti ei ole sääntöjen mukainen && salasanan pituus <= 0 -> Kirjaudu-buttonin className="cursor-not-allowed hover:text-white"
 
-# session ei päivity ollenkaan. Ehkä jonkinlainen kysymys, että haluaako pysyä kirjautuneena? Kuten julkisissapalveluissa (Kela, OmaKanta, Suomi.fi yms)
+# Rekistöröidessä kun rekistöröidy-painike on painettu -> tee napista semmoinen ettei voi painaa uudelleen / tee jonkinlainen latausindikaattori jotta käyttäjä tietää, että jotain tapahtuu. Tämä "ongelma" näkyy parhaiten Vercelin tarjoamassa testiympäristössä, jossa nopeudet eivät ole localhostin kaltaiset
+
+# Ehkä jonkinlainen kysymys, että haluaako pysyä kirjautuneena kun session on loppumassa? Kuten julkisissapalveluissa (Kela, OmaKanta, Suomi.fi yms)
```

Comment:
>session ei päivity ollenkaan

Tähän vain sen verran, että Lucia tosiaan päivittää sessionin. Päivitys tapahtuu AINA kun sessionin puoliväli on tavoitettu. 1h sessionin tapauksessa session päivitetään 30min kohdalla, jne.

Tässä vielä dokumentaatiosta pieni leike:

`More specifically, if the session expiration is set to 30 days (default), Lucia will extend the expiration by another 30 days when there are less than 15 days (half of the expiration) until expiration. `